.travis.yml

```diff
@@ -45,6 +45,8 @@ env:
     - RUN="unit"
     - RUN="unit-next" BOULDER_CONFIG_DIR="test/config-next"
     - RUN="coverage"
+    allow_failures:
```

Comment:
This shouldn't be indented I think?

Reply:
I think it should, following examples at https://github.com/travis-ci/travis-ci/issues/6661 and https://rnelson0.com/2015/11/25/allowing-expected-failures-with-travis-ci/.

Reply:
Hah, of course, silly of me to post that before looking at why Travis was failing... :(